opensciencegrid/default-build-config.json

```diff
@@ -0,0 +1,7 @@
+{
+    "standard_build": true,
+    "repo_build": false,
+    "base_os": ["el9"],
+    "osg_series": ["23"],
+    "base_repo": ["development", "testing", "release"]
+  }
```

Comment:
You're missing a newline here. I think this is something that VS Code just /does/. Maybe @jhiemstrawisc knows the configuration to set to force it to stop doing it

Reply:
I do! If you go to your settings explorer (on Mac, that's `Code-->Settings-->Settings`) you can search for "Insert Final Newline". Check the box to save the setting and you should be good to go.